okhttp/src/main/java/okhttp3/EventListener.java

```diff
@@ -38,7 +38,7 @@ public void dnsStart(Call call, String domainName) {
   }
 
   public void dnsEnd(Call call, String domainName, List<InetAddress> inetAddressList,
```

Comment:
Right, this will end up as `@Nullable`. I often favor doing the minimum amount of change to get a thing working, just haven't got to the failure case yet.